lib/pure/os.nim

```diff
@@ -1352,9 +1352,15 @@ elif defined(nintendoswitch):
   proc paramCount*(): int {.tags: [ReadIOEffect].} =
     raise newException(OSError, "paramCount is not implemented on Nintendo Switch")
 
+elif defined(genode):
+  proc paramStr*(i: int): TaintedString =
+    raise newException(OSError, "paramStr is not implemented on Genode")
```

Comment:
I will accept this, but I think it's not good to turn a clear compiletime error into a runtime error.